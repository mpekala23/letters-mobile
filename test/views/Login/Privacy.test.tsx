import * as React from 'react';
import { PrivacyScreen } from '@views';
import { render, toJSON } from '@testing-library/react-native';

describe('Terms of Service screen', () => {
  it('should match snapshot', () => {
    const { container } = render(<PrivacyScreen />);
    const tree = toJSON(container);
    expect(tree).toMatchSnapshot();
  });
});
