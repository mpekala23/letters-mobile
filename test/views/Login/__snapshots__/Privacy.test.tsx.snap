// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms of Service screen should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text>
      Hello world
    </Text>
  </View>
</View>
`;
